system-test/spanner.ts

```diff
@@ -39,7 +39,7 @@ import CreateBackupMetadata = google.spanner.admin.database.v1.CreateBackupMetad
 
 const PREFIX = 'gcloud-tests-';
 const RUN_ID = shortUUID();
-const LABEL = `gcloud-tests-${RUN_ID}`;
+const LABEL = `gcloud-system-tests-${RUN_ID}`;
```

Comment:
While we're changing this, could we go more distinct with something like "node-spanner-systests-"?

Reply:
That's cool. Thanks for doing that!